### DISCUSSION TITLE
Enhancing the Refinement Meeting Process

### DISCUSSION THREAD

**Discussion by qu1queee** [shipwright-io/community#253] [Ideas]
The Refinement meeting (happens every second Thursday ), needs a "refinement", some of the reasons:

- [1] Navigating the meeting issues is cumbersome
- [2] The way issues are refined, can be improved.
- [3] There is no clear outcome of the meeting, for example, is this meeting making a difference when shipping releases?
- [4] Low attendance.
- [5] Ownership of issues after refinement tends to be low. No strong commitment. 

This discussion aims to provide solutions to the above.

**Comment by qu1queee** [comment]
On [1], some efforts already took place, see [slack thread](https://kubernetes.slack.com/archives/C019ZRGUEJC/p1730379478332579)

**Comment by adambkaplan** [comment]
@qu1queee I'm resurrecting this discussion to get broader feedback from the community. Our backlog refinement process is effectively gone - the meeting has not been attended in quite some time, and of the few times it has happened I have often been the only one to attend.

I'd like concrete discussion and proposals to happen here in GitHub discussions - burying the details in a Slack thread adds friction and does not radiate decisions that were made. Things that I think we need to agree to:

- Meeting time - most of the community is based in Europe or Asia these days. I'd rather have the meeting happen at a time that is more friendly to those time zones. I personally can't attend at the current time slot.
- Goals of the meeting
- Ways that we communicate outcomes of the meeting (recording, transcript, Google Doc, etc.)
